stats/utils.py

```diff
@@ -0,0 +1,12 @@
+"""
+Misc utilities.
+"""
+
+
+def get_rowset_mapping(result_sets, required_fields):
+    """
+    Returns a list of mapped fields to the passed headers.
+    """
+
+    headers = result_sets['headers']
+    return list(map(lambda field, headers=headers: headers.index(field), required_fields))
```

Comment:
This logic looks clean.

Reply:
Purely, in the context of an academic discussion, imo, there is another way to achieve this as well 

`return [headers.index(field) for field in required_fields]`

ListComprehensions is always recommended as the fastest way to construct lists in Python. 

Imo, nothing wrong with what you have here, but it does have slightly more function calls than the LC way. Esp since you have to call list on the resulting map object.

It's strange, but I have never been a fan of Lambda expressions in Python. They always somehow felt .... out of place. I love the concept, but I have just never been a fan of the syntax. I have seen them in action in other languages like JS, but Python isn't a functional programming, and so my own experience with such patterns here is limited.

Reply:
I agree a list comprehension looks better. My go-to is usually a `map` in Erlang and after writing this it felt clunky. I'll switch it over.